Formula/ffmpeg.rb

```diff
@@ -111,15 +112,15 @@ def install
       --prefix=#{prefix}
       --enable-shared
       --enable-pthreads
-      --enable-gpl
       --enable-version3
       --enable-hardcoded-tables
       --enable-avresample
       --cc=#{ENV.cc}
       --host-cflags=#{ENV.cflags}
       --host-ldflags=#{ENV.ldflags}
     ]
-
+    
+    args << "--enable-gpl" if not build.without? "gpl"
```

Comment:
```ruby
args << "--enable-gpl" if build.with? "gpl"
```
Is the preferred syntax.

Reply:
Thanks, I was watching the Jenkins log output and saw the style check complaint. I've issued an update to the file.